torch/optim/nadam.py

```diff
@@ -260,13 +260,13 @@ def _single_tensor_nadam(params: List[Tensor],
             # Make autograd track the operations
             # by updating the grad and exp_avg directly and not using the
             # scalar "value" argument of addcdiv.
-            mu_product_next = mu_product * mu * mu_next
```

Comment:
This may be bc breaking...but I'm fairly certain it should not contain mu here since we already update mu_products in line 251

Reply:
I think this is clearly a bugfix yes.